### ISSUE TITLE
Add password reset link.

### ISSUE THREAD

**Issue by giorgiozoppi** [openMF/digital-bank-ui#42]
Rationale:
We want the user to be able to reset his credentials. Workflow:
1. The UI should provide a password reset button "Forgot password" 
2. The user click the password reset link and a email/sms has been sent.
3. The user click to the password reset link in the email and gets directed to a password reset page.
4. In the password reset page, he will be able to reset the password.

**Comment by RadheTians** [comment]
Hi @giorgiozoppi!!!

I would like to work on this potential issue so, can I please go ahead with it?

**Comment by giorgiozoppi** [comment]
Yes. We need as well the backend for this, using twilio. For the moment you can mock it. I might be able to prepare something today.

**Comment by RadheTians** [comment]
Thanks, @giorgiozoppi for your confirmation.

Kindly let me know once when you prepare that, thank you.

**Comment by giorgiozoppi** [comment]
In meanwhile go ahead with the password reset mail. Show us what you are able to do.

**Comment by RadheTians** [comment]
Sure @giorgiozoppi!! Kindly give me some time currently I am working on [tooltips](https://github.com/openMF/digital-bank-ui/issues/38)  issue, I get back to you soon.

**Comment by RadheTians** [comment]
Hi @giorgiozoppi sir!!

Here is the status of this issue. I was trying to build the backend for sending an email with a reset link in Node.js using **nodemailer** package but there are some security issues in my gmail account therefore I'm not able to move forward, kindly suggest to me any plan if you have.

![Screenshot from 2020-12-13 03-02-03](https://user-images.githubusercontent.com/50411704/101995538-a734b900-3cf0-11eb-9f61-c923a905466a.png)
![Screenshot from 2020-12-13 03-09-10](https://user-images.githubusercontent.com/50411704/101995541-ad2a9a00-3cf0-11eb-8eba-57b236fbef40.png)

**Comment by RadheTians** [comment]
Should I create a PR so that you can see my changes?
